Prog/BPK Summary Tabs_2014-18.sas

```diff
@@ -0,0 +1,1493 @@
+			PctBirthsPrenatIntrWht_2004 = births_prenat_intr_wht_2004/births_w_prenat_wht_2004;
+			PctBirthsPrenatIntrWht_2005 = births_prenat_intr_wht_2005/births_w_prenat_wht_2005;
+			PctBirthsPrenatIntrWht_2006 = births_prenat_intr_wht_2006/births_w_prenat_wht_2006;
+			PctBirthsPrenatIntrWht_2007 = births_prenat_intr_wht_2007/births_w_prenat_wht_2007;
+			PctBirthsPrenatIntrWht_2008 = births_prenat_intr_wht_2008/births_w_prenat_wht_2008;
+			PctBirthsPrenatIntrWht_2009 = births_prenat_intr_wht_2009/births_w_prenat_wht_2009;
+			PctBirthsPrenatIntrWht_2010 = births_prenat_intr_wht_2010/births_w_prenat_wht_2010;
+			PctBirthsPrenatIntrWht_2011 = births_prenat_intr_wht_2011/births_w_prenat_wht_2011;
+			PctBirthsPrenatIntrWht_2012 = births_prenat_intr_wht_2012/births_w_prenat_wht_2012;
+			PctBirthsPrenatIntrWht_2013 = births_prenat_intr_wht_2013/births_w_prenat_wht_2013;
+			PctBirthsPrenatIntrWht_2014 = births_prenat_intr_wht_2014/births_w_prenat_wht_2014;
+			PctBirthsPrenatIntrWht_2015 = births_prenat_intr_wht_2015/births_w_prenat_wht_2015;
+			PctBirthsPrenatIntrWht_2016 = births_prenat_intr_wht_2016/births_w_prenat_wht_2016;
+
+			PctBirthsPrenatIntrAsn_2003 = births_prenat_intr_asn_2003/births_w_prenat_asn_2003;
+			PctBirthsPrenatIntrAsn_2004 = births_prenat_intr_asn_2004/births_w_prenat_asn_2004;
+			PctBirthsPrenatIntrAsn_2005 = births_prenat_intr_asn_2005/births_w_prenat_asn_2005;
+			PctBirthsPrenatIntrAsn_2006 = births_prenat_intr_asn_2006/births_w_prenat_asn_2006;
+			PctBirthsPrenatIntrAsn_2007 = births_prenat_intr_asn_2007/births_w_prenat_asn_2007;
+			PctBirthsPrenatIntrAsn_2008 = births_prenat_intr_asn_2008/births_w_prenat_asn_2008;
+			PctBirthsPrenatIntrAsn_2009 = births_prenat_intr_asn_2009/births_w_prenat_asn_2009;
+			PctBirthsPrenatIntrAsn_2010 = births_prenat_intr_asn_2010/births_w_prenat_asn_2010;
+			PctBirthsPrenatIntrAsn_2011 = births_prenat_intr_asn_2011/births_w_prenat_asn_2011;
+			PctBirthsPrenatIntrAsn_2012 = births_prenat_intr_asn_2012/births_w_prenat_asn_2012;
+			PctBirthsPrenatIntrAsn_2013 = births_prenat_intr_asn_2013/births_w_prenat_asn_2013;
+			PctBirthsPrenatIntrAsn_2014 = births_prenat_intr_asn_2014/births_w_prenat_asn_2014;
+			PctBirthsPrenatIntrAsn_2015 = births_prenat_intr_asn_2015/births_w_prenat_asn_2015;
+			PctBirthsPrenatIntrAsn_2016 = births_prenat_intr_asn_2016/births_w_prenat_asn_2016;
+
+			PctBirthsPrenatIntrBlk_2003 = births_prenat_intr_blk_2003/births_w_prenat_blk_2003;
+			PctBirthsPrenatIntrBlk_2004 = births_prenat_intr_blk_2004/births_w_prenat_blk_2004;
+			PctBirthsPrenatIntrBlk_2005 = births_prenat_intr_blk_2005/births_w_prenat_blk_2005;
+			PctBirthsPrenatIntrBlk_2006 = births_prenat_intr_blk_2006/births_w_prenat_blk_2006;
+			PctBirthsPrenatIntrBlk_2007 = births_prenat_intr_blk_2007/births_w_prenat_blk_2007;
+			PctBirthsPrenatIntrBlk_2008 = births_prenat_intr_blk_2008/births_w_prenat_blk_2008;
+			PctBirthsPrenatIntrBlk_2009 = births_prenat_intr_blk_2009/births_w_
```

Comment:
We should inflation adjust these income amounts to 2018 dollars for the table output. You can use the %dollar_convert() macro to do this.

```
%dollar_convert( medianmrtginc1_4m_2009, r_medianmrtginc1_4m_2009, 2009, 2018 )
%dollar_convert( medianmrtginc1_4m_2010, r_medianmrtginc1_4m_2010, 2010, 2018 )
etc.
```